xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_game_types.h

```diff
@@ -419,7 +419,7 @@ typedef struct AddonProps_Game
   /*!
    * Number of proxy DLL paths provided.
    */
-  unsigned int proxy_dll_count;
+  size_t proxy_dll_count;
```

Comment:
Can you drop these? I've fixed them in a recent commit that I'll PR soon.

BTW, these would require `#include <stdint.h>`

Reply:
For reference: https://github.com/garbear/xbmc/commit/25d5bf9

Reply:
yep I'll drop them.

Reply:
Thanks. I'll get that commit in Kodi in the next few weeks.